### PR TITLE
Fix #323: Adds note about CLI default download location

### LINKED ISSUE #323: Add note about default download location of client binaries in README

### PR DESCRIPTION
**Pull request by navidshaikh**
Fixes #323

### CODE REVIEW

**Comment by budhrg** [comment]
LGTM any thoughts @hferentschik ?

**Comment by brgnepal** [comment]
LGTM any thoughts @hferentschik ?

File: README.adoc [hunk 1]
```diff
@@ -159,6 +159,9 @@ The possible options are:
   * `--path`: Download the binary to the specified path on the host.
 +
 The supported options for `service` are: Docker, OpenShift.
++
+Client binaries are downloaded in `data/service-manager/bin/<service>/<cli-version>/` directory under Vagrant home directory by default.
```
Comment (hferentschik) [inline README.adoc:163]: "Client binaries are per default downloaded to `$VAGRANT_HOME/data/service-manager/bin/<service>/<cli-version>/`. Per default `$VAGRANT_HOME` is `.vagrant.d` in your home directory.

**Comment by hferentschik** [comment]
Merged in a slightly different form

**Pull request closed without merging**